sourcesheets/views.py

```diff
@@ -1020,7 +1020,7 @@ def sheets_by_ref_api(request, ref):
     """
     API to get public sheets by ref.
     """
-    return jsonResponse(get_sheets_for_ref(ref))
+    return jsonResponse(get_sheets_for_ref(ref, include_collections=True))
```

Comment:
I think this API should take a URL parameter that indicates whether it should include collections. For boolean parameters, our standard is to use 0 to mean false and 1 to mean true. Here's an example ```bool(int(request.GET.get("annotate", 0)))```